content/conditions/cellulitis/content-7.md

```diff
@@ -0,0 +1,11 @@
+## Call 999 or go to A&E now if you have cellulitis with:

```

Comment:
Good spot. Will make sure it's added in all files that include that abbreviation. We should also find a better way to parse common abbreviations. Something more globally.